server/repository/src/migrations/v2_00_00/assets/reference_data.rs

```diff
@@ -27,7 +27,7 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
     sql!(
         connection,
         r#"
-        CREATE TABLE asset_type (
+        CREATE TABLE asset_catalogue_type (
```

Comment:
possibly a comment to say that there was a clash with an existing table `asset_type` and that's why this is called `asset_catalogue_type` which doesn't match the object names etc.
It's likely not going to be helpful long term, but just in case the question comes up.